src/components/SelectBox.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { ControllerRenderProps } from 'react-hook-form';
+import styled from 'styled-components';
+
+import DropdownBlock from './DropdownBlock';
+import { Icon24 } from '../shared/components/Icon';
+
+import theme from '../styles/theme';
+import Spinner from '../shared/components/Spinner';
+
+export type Option = {
+  label: string; // Real Value
+  value: string; // System Value
+};
+
+interface SelectBox {
+  status?: 'active' | 'loading' | 'disabled' | 'error';
+  placeholder?: string;
+  options: Option[];
+  optionSize?: 'small' | 'large';
+  width?: number;
+}
+interface SelectBoxWithControllerField extends ControllerRenderProps, SelectBox {}
+
+const SelectBox = React.forwardRef<HTMLDivElement, SelectBoxWithControllerField>(
+  ({
+    status = 'active',
+    placeholder: declaredPlaceHolder = 'Select',
+    options,
+    optionSize = 'small',
+    width = 250,
+    onChange,
+    value,
+  }) => {
```

Comment:
```suggestion
const SelectBox = React.forwardRef<HTMLDivElement, SelectBoxWithControllerField>(
  ({
    status = 'active',
    placeholder: declaredPlaceHolder = 'Select',
    options,
    optionSize = 'small',
    width = 250,
    onChange,
    value,
  }, ref) => {
```
forwardRef로 생성되었지만 ref를 사용하지 않고 있습니다.